### DISCUSSION TITLE
Take control gives 502 gateway error

### DISCUSSION THREAD

**Discussion by Shidougit** [amidaware/tacticalrmm#1648] [General]
**Server Info (please complete the following information):**
 - OS: Debian 10
 - Browser: chrome
 - RMM Version (as shown in top left of web UI): 0.16.5

**Installation Method:**
  - [x ] Standard
  - [ ] Docker

**Agent Info (please complete the following information):**
- Agent version (as shown in the 'Summary' tab of the agent from web UI): 2.5.0
- Agent OS: Server 2012 R2

**Describe the bug**
Since the last update the agent returns a 502 gateway error when trying to take control of the client.

**To Reproduce**
1. Got to any server.
2. Right click on a server en select Take Control.
3. A new windows opens with the 502 error.

**Expected behavior**
A black screen with two buttons in the top left connect and RDP connect.

**Screenshots**
<img width="829" alt="image" src="https://github.com/amidaware/tacticalrmm/assets/139108954/31c3e72d-3df3-4f3d-95b2-29c407f414f2">

**Comment by dinger1986** [comment]
meshcentral service most likely hasnt started,

Please run https://docs.tacticalrmm.com/troubleshooting/#server-troubleshooting-script or https://docs.tacticalrmm.com/troubleshooting/#all-other-errors

**Reply by Shidougit** [reply]
Yes it was indeed the meshcentral. It couldn't install the mongodb with npm. I manually installed it for hem and then the service started again and all is well :)

**Reply by dinger1986** [reply]
great
